Dockerfile.build

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ARG BASE=golang:1.13-alpine
+ARG BASE=golang:1.15.2-alpine
```

Comment:
I don't thing we want to ping the patch. I verified that current `golang:1.15` has Go 1.15.2.